src/app/register/page.tsx

```diff
@@ -0,0 +1,220 @@
+"use client";
+import { useState } from "react";
+import { useSignUp } from "@clerk/nextjs";
+import { useRouter } from "next/navigation";
+
+const RegisterPage = () => {
+  const { isLoaded, signUp, setActive } = useSignUp();
+  const [name, setName] = useState("");
+  const [description, setDescription] = useState("");
+  const [website, setWebsite] = useState("");
+  const [clerkID, setClerkID] = useState("");
+  const [logo, setLogo] = useState("");
+  const [reimbursements, setReimbursements] = useState([]);
+  const [status, setStatus] = useState("");
+  const [pendingVerification, setPendingVerification] = useState(false);
+  const [code, setCode] = useState("");
+  const router = useRouter();
```

Comment:
the library i mentioned is called react-hook-form, it makes managing state like this a lot easier if you want to take a look